doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -426,8 +430,9 @@ Standard IO
 The Ltac2 language features non-backtracking IO, notably mutable data and
 printing operations.
 
-Mutable fields of records can be modified using the set syntax. Likewise,
-built-in types like `string` and `array` feature imperative assignment. See
+Mutable fields of records can be modified using the syntax
+:n:`@ltac2_expr0 .( @qualid ) := @ltac2_expr5`. Likewise, built-in
+types like `string` and `array` feature imperative assignment. See
 modules `String` and `Array` respectively.
```

Comment:
This paragraph seems out of place.  I would think about deleting it entirely since your other change describes modifying/accessing records.